internal/host/hostcommands/install_cmd.go

```diff
@@ -192,22 +192,35 @@ func (i *installCmd) getFullInstallerCommand(cluster *common.Cluster, host *mode
 
 func (i *installCmd) getProxyArguments(clusterName, baseDNSDomain, httpProxy, httpsProxy, noProxy string) []string {
 	cmd := make([]string, 0)
+	if httpProxy == "" && httpsProxy == "" {
+		return cmd
+	}
 
-	if httpProxy != "" || httpsProxy != "" {
-		if httpProxy != "" {
-			cmd = append(cmd, "--http-proxy", httpProxy)
-		}
-		if httpsProxy != "" {
-			cmd = append(cmd, "--https-proxy", httpsProxy)
-		}
+	if httpProxy != "" {
+		cmd = append(cmd, "--http-proxy", httpProxy)
+	}
 
+	if httpsProxy != "" {
+		cmd = append(cmd, "--https-proxy", httpsProxy)
+	}
+
+	noProxyTrim := strings.TrimSpace(noProxy)
+	if noProxyTrim == "*" {
+		cmd = append(cmd, "--no-proxy", noProxyTrim)
```

Comment:
this can be `cmd = append(cmd, "--no-proxy *")`

Reply:
I'm not sure, because the function returns a slice, and each individual element of it is later quoted in https://github.com/openshift/assisted-service/blob/c8566575a0fb212f91b398205cffb3f41037f67a/internal/host/hostcommands/install_cmd.go#L190. So I'll get `'--no-proxy *'` (which is not a valid command option), instead of `'--no-proxy' '*'`.